vars/fcosKola.groovy

```diff
@@ -24,8 +26,8 @@ def call(params = [:]) {
         arch = "--arch=${arch}"
     }
 
-    // Create a unique output directory for this run of fcosKola
-    def outputDir = shwrapCapture("mktemp -d ${cosaDir}/tmp/fcosKola-XXXXX")
```

Comment:
Hmm, though then if the caller does parallel calls without passing markers, we'll be back to clobbering results, no? (Not only the published artifacts, but we'd also be outputting artifacts which will be an amalgamation of multiple runs.)

Maybe simplest is to default the marker to some generated value? E.g. using `uuidgen | cut -f1 -d-`. At least from the log output they could still match up with parallel branches.

Reply:
Made a new upload and updated the commit title/message (along with title and description of this PR) to be more appropriate.